SentinelOne/sentinelone/_meta/fields.yml

```diff
@@ -1,9 +1,14 @@
 # meta fields (about agent gathering events)
-        
+
 sentinelone.eventid:
   name: sentinelone.eventid
   type: long
   description: ""
+
+host.ip:
+  name: host.ip
+  type: array
+  description: ""
```

Comment:
Why declare a standard ECS field in the sentinel one fields ?

Reply:
Ingest is not treating this field as an array if if don't specify it in this file =/

Reply:
Field was removed in commit https://github.com/SEKOIA-IO/intake-formats/pull/75/commits/ea3a517c4c891dfa176885fd7045639cd957a8ca